src/ds3.c

```diff
@@ -453,6 +453,23 @@ void ds3_request_set_max_keys(ds3_request* _request, uint32_t max_keys) {
     _set_query_param(_request, "max-keys", max_keys_s);
 }
 
+const char* UNSIGNED_LONG_BASE_10 = "4294967296";
```

Comment:
I would make these static so that they are not exported in the symbol table and stay local to the file.